src/lib/components/Sidebar.svelte

```diff
@@ -45,7 +45,10 @@ The sidebar/drawer.
         <label for={drawerId} class="dy-drawer-overlay" />
         <ul class="dy-menu p-1 overflow-y-auto w-3/4 sm:w-80 bg-base-100 text-base-content">
             <!-- Sidebar content here -->
-            <li><a href="/"><BibleIcon /><span>{$globalConfig.name}</span></a></li>
+            <a class="fill" href="/">
```

Comment:
What happens if there is no nav_drawer.png or nav_drawer@2x.png specified?

Reply:
This is what happens:

![image](https://user-images.githubusercontent.com/517814/201806463-55953bae-12df-449f-aaad-fb7db8263218.png)

Reply:
I like having an icon and the App Name if there is no Drawer Image.  Do you want me to try to get the App Icon or just use the icon we were using before?

Reply:
I talked it over with another developer and he said the back end should make sure there is a file.  So this is OK.